hex_blueprints/exit_codes.py

```diff
@@ -0,0 +1,18 @@
+
+## error codes 
+EXIT_CODE_BAD_REQUEST = 206
+EXIT_CODE_INVALID_PROJECT_ID = 201
+EXIT_CODE_INVALID_RUN_ID = 202
+EXIT_CODE_EXCESSIVE_REQUESTS = 203
+EXIT_CODE_HEX_SERVER_ERROR = 204
+EXIT_CODE_AUTHENTICATION_ERROR = 205
+EXIT_CODE_UNKNOWN_ERROR = 3
+
+## run status codes
+## these are provided here: https://learn.hex.tech/docs/develop-logic/hex-api/api-reference#operation/GetRunStatus
+EXIT_CODE_PENDING = 220
+EXIT_CODE_RUNNING = 221
+EXIT_CODE_ERRORED = 222
+EXIT_CODE_COMPLETED = 0 ## success
+EXIT_CODE_KILLED = 224
+EXIT_CODE_UNABLE_TO_ALLOCATE_KERNEL = 225
```

Comment:
Let's rename these to HEX_CODE_Xxx or the like. I don't want exit codes for the script itself to have names conflating with hex codes. 
Or are both sets of these all exit code values from our script?

Reply:
@ericelsken They're all exit codes specifically for our script, not provided by Hex. The consistent naming right now would still be `EXIT_CODE_{REASON}`

Reply:
Okay. That was my second guess. Any reason they aren't all grouped together? [](edited-with-Axolo)

Reply:
We started doing a new system of grouping them by “cause”.

So bottom section is specifically for checking run status. The top section is specifically for the issues with the API. [](edited-with-Axolo)